src/rust/engine/src/externs/mod.rs

```diff
@@ -164,6 +164,15 @@ pub fn store_bool(val: bool) -> Value {
   Value::from(val.to_py_object(gil.python()).into_object())
 }
 
+///
+/// Check if a Python object has the specified field.
+///
+pub fn hasattr(value: &Value, field: &str) -> bool {
+  let gil = Python::acquire_gil();
+  let py = gil.python();
+  value.hasattr(py, field).expect("")
```

Comment:
Seeing `.expect("")` worries me. (1) Is there any reason we wouldn't just `.unwrap()`? (2) The docs here (https://docs.rs/cpython/0.5.1/cpython/type.PyResult.html) have no information about when this might fail and because of that I might want to say something like "hasattr failed" here unless we can trust the rust panic traceback (which I *think* I recall not always obviously pointing to the failing line?).

Reply:
Oh, I meant to finish this! Oops.